guides/catalog/02 Creating and Managing Products.ipynb

```diff
@@ -156,120 +131,140 @@
+    "\n",
+    "See the [`Bands` Documentation page](https://docs.descarteslabs.com/descarteslabs/catalog/docs/band.html#bands) for more information about properties and other band types not covered in this example."
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "c5da3745-c0b9-4ade-b291-834a3f4b8129",
+   "id": "587516dc-1472-4771-bb1e-8a25c8c09b18",
    "metadata": {},
    "source": [
-    "## Creating images\n",
-    "There are two general mechanisms of creating images in the catalog. Upload is the primary mechanism for creating images, either by uploading supported image files types such as GeoTIFF or JPEG, or by uploading image data in the form of a numpy ndarray. The other mechanism is to create “remote” image entries in the catalog without supplying the actual image data."
+    "## Uploading Images - File Paths\n",
+    "`image.upload()` uploads imagery to a `Product` via a list of specified file paths. This returns an `ImageUpload`. Images are uploaded and processed asynchronously, so they are not available in the catalog immediately. With `upload.wait_for_completion()` we wait until the upload is completely finished:"
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "3f2f9cdb-70d3-4bd2-8c56-4a76a4dc544a",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e7fde391-1842-42d3-a7a7-146f3403f7e3",
    "metadata": {
     "tags": []
    },
+   "outputs": [],
    "source": [
-    "### Uploading image files\n",
-    "If your data already exists on disk as an image file, usually a GeoTIFF or JPEG file, you can upload it directly.\n",
-    "\n",
-    "In the following examples we will upload data to the single band we created representing the blue light spectrum. "
+    "# Set any attributes that should be set on the uploaded images\n",
+    "image = Image(product=product, name=\"image1\")\n",
+    "# Set acquired date\n",
+    "image.acquired = \"2023-06-15\"\n",
+    "# Set metadata\n",
+    "image.cloud_fraction = 0.1\n",
+    "# Do the upload\n",
+    "image_path = \"data/rgb.tif\"\n",
+    "upload = image.upload(image_path)\n",
+    "upload.wait_for_completion()\n",
+    "upload.status"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "587516dc-1472-4771-bb1e-8a25c8c09b18",
+   "id": "ac4353fc-4b35-4fff-9479-40314a5f3928",
    "metadata": {},
    "source": [
-    "Now you create a new image and use `image.upload()` to upload imagery to the new product. This returns a ImageUpload. Images are uploaded and processed asynchronously, so they are not available in the catalog immediately. With `upload.wait_for_completion()` we wait until the upload is completely finished."
+    "### _Note: Check your Results!_\n",
+    "You can now search and retrieve this `Image` in either [Explorer](app.descarteslabs.com/explorer) or `Catalog API` directly:"
```

Comment:
I think this Explorer link needs to have an `https://` at the front of it.

In my notebook, this gets linked to the following, which suggests that Jupyter thinks this is a local file:
```
https://app.descarteslabs.com/workbench/user/jay@descarteslabs.com/files/cmoulton_notebooks/guides/catalog/app.descarteslabs.com/explorer?_xsrf=2%7Ccafe4b75%7C19d28d19d7050124373529a2b96d77ca%7C1698690843
```